flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -1473,6 +1473,14 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "EXTRACT(DOY FROM f16)",
       "315")
 
+    testSqlApi(
+      "EXTRACT(DOW FROM f18)",
+      "1")
+
+    testSqlApi(
+      "EXTRACT(DOW FROM f16)",
```

Comment:
Can you add a test (maybe in `ScalarFunctionsValidationTest`?) for checking the behavior on a `TIME` type?